<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationWindow</class>
 <widget class="QMainWindow" name="VisualizationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[Title]</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="dateRangeLabel">
      <property name="text">
       <string>[Date Range]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="plotDataSelect"/>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="plotTypeSelect">
      <item>
       <property name="text">
        <string>Bar Graph</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scatter Plot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pie Graph</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="plotButton">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="changeVisLabel">
      <property name="text">
       <string>Change Visualization:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="memberNameLabel">
      <property name="text">
       <string>[Member Name]</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="visualization" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>vis2</zorder>
         <zorder>vis2</zorder>
         <zorder>vis2</zorder>
         <zorder>vis2</zorder>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="PieChartWidget" name="vis2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <zorder>visualization</zorder>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <action name="actionPrint">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Print...</string>
    </property>
   </action>
   <action name="actionClose">
    <property name="text">
     <string>Close</string>
    </property>
   </action>
   <addaction name="fileMenu"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../external/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PieChartWidget</class>
   <extends>QWidget</extends>
   <header>../src/ui/PieChartWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
